cmd/install/assets/hypershift_operator.go

```diff
@@ -778,6 +778,11 @@ func (o HyperShiftOperatorClusterRole) Build() *rbacv1.ClusterRole {
 				Resources: []string{"*"},
 				Verbs:     []string{"*"},
 			},
+			{
+				APIGroups: []string{"image.openshift.io"},
+				Resources: []string{"*"},
+				Verbs:     []string{"*"},
+			},
```

Comment:
Also please add imagestreams to the list of resources to dump:
https://github.com/openshift/hypershift/blob/a50f62208e677dacb234f8c46c3ea64abb0dc4bb/cmd/cluster/core/dump.go#L180-L209